tests/integ/modin/groupby/test_groupby_basic_agg.py

```diff
@@ -364,6 +450,38 @@ def test_groupby_agg_on_groupby_columns(
     )
 
 
+@pytest.mark.parametrize(
+    "by", ["col1", ["col1", "col2", "col3"], ["col1", "col1", "col2"]]
+)
+@pytest.mark.parametrize("as_index", [True, False])
+@pytest.mark.parametrize("sort", [True, False])
+def test_groupby_agg_on_groupby_columns_named_agg(
+    basic_snowpark_pandas_df, by, as_index, sort
+) -> None:
+    query_count = 2
+    kwargs = {}
+    if by == ["col1", "col1", "col2"] and not as_index:
+        kwargs = {
+            "expect_exception": True,
+            "expect_exception_type": ValueError,
+            "expect_exception_match": "cannot insert col1, already exists",
```

Comment:
from here: https://github.com/snowflakedb/snowpandas/pull/982#discussion_r1569317244

yes, please open a pandas issue and put a link to it in a comment here.

Reply:
Looks like the error is because the previous test (groupby_agg with no namedagg) has two aggregations on `col1`, while this only has 1. Even with two aggregations on `col1` this still fails though, so I've opened an issue on the pandas repo!